### ISSUE TITLE
Proposal: What to do about yt-dlps new nightly branch?

### ISSUE THREAD

**Issue by vxbinaca** [bibanon/tubeup#281]
I would like to solicit thoughts from other devs on the new [nightly builds flag](https://github.com/yt-dlp/yt-dlp/releases/tag/2023.03.03) that are canary builds on a per-commit basis with yt-dlp. On the one hand this is a great way to get users of Tubeup to get fixes to things fast, so we don't have to wait 10 hours for regex fixes in extractors. On the other hand it breaks things.

Should nightlys be turned on by default in the configs for Tubeup, or should we prize stability?

What are your thoughts on the new canary feature?

**Comment by vxbinaca** [comment]
Stability by default sounds good now that I think about it. A user could easily be instructed to or on their own switch branches.

**Comment by mrpapersonic** [comment]
Forcing nightlys by default is a double edged sword. Every once in a while you get a broken extractor in a stable release, but that typically doesn't happen often and it would probably happen much more often while on unstable releases. (Plus those releases aren't even posted on PyPI, so I don't see a simple way of pulling that off anyway)

**Comment by vxbinaca** [comment]
Good point. So if broken extractor, would you instruct a issue submitter to check yt-dlps github then switch to the canary branch?

The nice part about that is we don't need to implement this on our end. The user can just switch branches.
